jena-arq/src/main/java/org/apache/jena/sparql/core/mem/OrderedTupleTable.java

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
The license header is differently formatted and in fact the original headers on some files have been reformatted.

Please use the same, standard header and layout.